tests/python/unittest/test_sparse_operator.py

```diff
@@ -322,17 +322,15 @@ def lt(l, r):
     def le(l, r):
         return check_all(l, r, lambda a, b: a <= b)
 
-    def least_sparse(lstype, rstype):
-        if lstype == 'default' and rstype == 'default':
+    def elemwise_mul_stype(lstype, rstype):
+        if lstype == rstype:
+            return lstype
+        elif lstype == 'default' and rstype == 'row_sparse':
+            return 'row_sparse'
+        elif rstype == 'default' and lstype == 'row_sparse':
```

Comment:
just a nit, will "elif lstype == 'row_sparse' and rstype == 'default'" easier to read?